### PR TITLE
bubble streams / damage tiles

### PR DESCRIPTION
**Pull request by derekjbell**

### CODE REVIEW

File: Assets/Scripts/GivePlayerDamage.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GivePlayerDamage : MonoBehaviour
+{
+    public int damageAmount = 10;
+    public float cooldownTime = 2.0f;
+    private float nextDamageTime = 0.0f;
+
+    void OnTriggerEnter2D(Collider2D other)
```
Comment (HBurd) [inline Assets/Scripts/GivePlayerDamage.cs]: if there's a cooldown shouldn't this be an `OnTriggerStay2D` so you can't just wait in the spikes?

Comment (derekjbell) [reply Assets/Scripts/GivePlayerDamage.cs]: that's how I wanted it if I'm understanding you correctly so you can have a second to sit in the spikes and not immediately get damaged again

Comment (derekjbell) [reply Assets/Scripts/GivePlayerDamage.cs]: unless that means that it will never trigger again I see what you're saying

Comment (derekjbell) [reply Assets/Scripts/GivePlayerDamage.cs]: then yeah could change it over to `OnTriggerStay2D`

**Pull request merged**